tapchannel/allocation_sort.go

```diff
@@ -43,7 +43,7 @@ func (s sortableAllocationSlice) Swap(i, j int) {
 }
 
 // Less is a modified BIP69 output comparison, that sorts based on value, then
-// pkScript, then CLTV value.
+// pkScript, then CLTV value and finally HtlcIndex.
 //
 // NOTE: Part of the sort.Interface interface.
 func (s sortableAllocationSlice) Less(i, j int) bool {
```

Comment:
non-blocking nit: AFAICT the stdlib recommendation now is to use `slices.SortFunc` instead of `sort.Sort`.

Reply:
Makes sense. Also found a way to simplify the code quite a bit.